sdks/java/core/src/main/java/org/apache/beam/sdk/transforms/reflect/DoFnInvoker.java

```diff
@@ -135,6 +135,9 @@
     /** Provide a reference to the input element. */
     InputT element(DoFn<InputT, OutputT> doFn);
 
+    /** Provide a reference to the input sideInput. */
```

Comment:
to the side input with the specified tag.

Reply:
done